### PR TITLE
Cmdclass install doesn't run on pip install. 

### PR DESCRIPTION
**Pull request by artyaltanzaya**
- Added troubleshooting section on detectron2
- Updated pypi package version

### CODE REVIEW

**Comment by artyaltanzaya** [comment]
Changes made
1. Moved cmd class custom installation commands to be installed as a separate script. 
2. Required dependencies will install when users import the packages.

File: autodistill_detic/__init__.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 from autodistill_detic.detic_model import DETIC
 
-__version__ = "0.1.1"
```
Comment (capjamesg) [inline autodistill_detic/__init__.py:3]: I think the version should be `0.1.2`?

**Review by capjamesg — COMMENTED** [review]
Quick comment to review. Other than that, looks good!

File: autodistill_detic/__init__.py (hunk 1 above)
Comment (artyaltanzaya) [reply autodistill_detic/__init__.py:3]: latest is 0.1.3 which is added

**Pull request merged**